pdf-rendering-template/bin/render-pdf.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * node.js script to render a given HTML file into the given PDF filename.
+ * Details of the API are at https://developers.google.com/web/tools/puppeteer/
+ *
+ * This command is invoked as:
+ *
+ *   $ node bin/render-pdf.js input.html output.pdf
+ *
+ */
+
+const puppeteer = require("puppeteer");
+
+const inputPath = `file://${process.argv[2]}`;
+const outputPath = process.argv[3];
+
+console.log("Input path:", inputPath);
+console.log("Output path:", outputPath);
+
+(async () => {
+  const browser = await puppeteer.launch();
+  const page = await browser.newPage();
+
+  console.log("Starting PDF conversion");
+  await page.goto(inputPath, { waitUntil: "networkidle2" });
+
+  // https://github.com/GoogleChrome/puppeteer/blob/v1.4.0/docs/api.md#pagepdfoptions
+  await page.pdf({
+    path: outputPath,
+    format: "A4"
+  });
+
+  console.log("PDF conversion complete");
+  await browser.close();
+})();
```

Comment:
```suggestion
})().catch(error => {
  console.error(error);
  
  process.exit(1);
});
```

Unhandled promise rejections won't cause Node to exit with an error code, and will print an ugly warning:

```
❯ node -e '(async () => { throw new Error(); })()'
(node:6668) UnhandledPromiseRejectionWarning: Error
    at [eval]:1:22
    at [eval]:1:37
    at Script.runInThisContext (vm.js:131:18)
    at Object.runInThisContext (vm.js:295:38)
    at Object.<anonymous> ([eval]-wrapper:10:26)
    at Module._compile (internal/modules/cjs/loader.js:1200:30)
    at evalScript (internal/process/execution.js:98:25)
    at internal/main/eval_string.js:23:3
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6668) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:6668) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

vs: 
```
❯ node -e '(async () => { throw new Error(); })().catch(e => { console.error(e); process.exit(1) })'
Error
    at [eval]:1:22
    at [eval]:1:37
    at Script.runInThisContext (vm.js:131:18)
    at Object.runInThisContext (vm.js:295:38)
    at Object.<anonymous> ([eval]-wrapper:10:26)
    at Module._compile (internal/modules/cjs/loader.js:1200:30)
    at evalScript (internal/process/execution.js:98:25)
    at internal/main/eval_string.js:23:3
```